broken_link_checker/checker.py

```diff
@@ -25,25 +23,20 @@ class Checker:
         just verify the availability of these URL
     """
 
-    def __init__(self, host: str, delay: int = 1, deep_scan=False):
+    def __init__(self, host: str, delay: int = 1, deep_scan: bool = False):
         """Init the checker."""
         # We config the logger
         self.logging = logging.getLogger(f'checker({host})')
         self.logging.setLevel(logging.DEBUG)
         self.logging.debug('We initialize the checker for %s' % host)
 
         # We config the connection
-        self.conn = urllib3.connection_from_url(
-            host,
-            # We config the timeout
-            timeout=Timeout(connect=2.0, read=7.0),
-            headers=urllib3.util.make_headers(
-                user_agent="BrokenLinkChecker/1.0",
-                keep_alive=True
-            ),
-            # We config the max number of connection
-            maxsize=1,
-        )
+        self.conn = requests.session()
+        self.headers = {
+            "user_agent": "BrokenLinkChecker/1.0",
```

Comment:
I think we should add a valid/common user agent here, IMO like Mozilla... etc what do you think ?
And there is a typo, it should be `"User-agent"` not `"user_agent"`
```suggestion
            "User-Agent": "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:32.0) Gecko/20100101 Firefox/32.0",
            "Host": host,
            "Pragma": "no-cache",
            "Connection": "keep-alive",
            "Accept": "*/*",
            "Accept-Encoding": "gzip,deflate,sdch",
            "Cache-Control": "no-cache"
```

Reply:
The idea was that in case of high traffic in the server, the user should be able to differentiate traffic from the broken_link_checker and traffic from another application.